### DISCUSSION TITLE
【灌水】我家邦布最近好像交了什么奇怪的朋友

### DISCUSSION THREAD

**Discussion by Huanshenxuanlong** [share121/inter-knot#3191] [其他]
我家邦布最近老是早上出去晚上回来，特别勤，今天我跟出去一看看到它和他的朋友围着一只小一圈脖子上带着橘色围巾上面写着“01”的邦布，好像在玩什么，有谁知道那只邦布是谁家的，之前我好像没看到过啊。

**Comment by Huanshenxuanlong** [comment]
那只邦布的型号没见过，看起来和现在型号的邦布不一样，是什么老型号还是没出的新型号啊

**Comment by Manbow1997** [comment]
好像是六分街卖片兄妹家的，下次你跟上去看看
